### DISCUSSION TITLE
Additional versions for documentation site

### DISCUSSION THREAD

**Discussion by wesfincher** [primefaces/community#1178] [PrimeVue]
Currently only the latest major versions are available for the documentation site. Because of current breaking changes (mine are related to Bootstrap), I need to access prior versions of documentation.  Ideally, a complete list of versions would be available for anyone stuck on any version for any reason

**Comment by brianchristensen** [comment]
I have the same problem in my application.  We need access to the v6 documentation, would you please provide?  Complete list of versions available would be even better.
